Makefile

```diff
@@ -46,12 +56,10 @@ swagger_gen: $(SWAGGER)
 #   Tools   #
 #############
 
-GOLINT = $(GO_BIN_DIR)/golint
-MOCKGEN = $(GO_BIN_DIR)/mockgen
-CHANGIE = $(GO_BIN_DIR)/changie
-GORELEASER = $(GO_BIN_DIR)/goreleaser
 YQ = docker run --rm -v $(shell pwd):/workdir mikefarah/yq:4
 
+tools: $(GOLINT) $(MOCKGEN) $(CHANGIE) $(GORELEASER)
```

Comment:
I think this duplicates the `install_tools` recipe apart from the `.PHONY`. Is there a reason you added this one?

Reply:
Yeah, sorry, this is from something else I was working on, included my mistake.